src/views/v2/events-bar/views.php

```diff
@@ -13,22 +13,29 @@
  *
  */
 use Tribe\Events\Views\V2\Manager;
+use Tribe\Events\Views\V2\View;
 
 $public_views = tribe( Manager::class )->get_publicly_visible_views();
+$view_instance = View::make( $this->get( 'view_class_name' ) );
```

Comment:
I feel like at some point if these vars can come from the backend and we document them on the docblock, the template will be way cleaner. Maybe we should discuss this with the BE devs.